doc/manuals/user/context_providers.md

```diff
@@ -159,7 +131,6 @@ Some additional comments:
     the CPr results before responding them back to client. An issue corresponding to this limitation has been created:
```

Comment:
On above line I have found a NGSIv1 mention that should be fixed:

```
First, Orion doesn't include the filter in the `POST /v1/queryContext` operation forwarded to CPr
```

should be changed to:

```
First, Orion doesn't include the filter in the `GET /v2/entities` operation forwarded to CPr
```

Reply:
Fixed in 138fbfa943a2aec7992c0c84422d254a2c2f319f